modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -6090,6 +6091,21 @@ public static String classNameToResourceName(String clsName) {
         return clsName.replaceAll("\\.", "/") + ".class";
     }
 
+    /**
+     * Gets a name of class by it resource name.
+     *
+     * @param rsrcName Resource name.
+     * @return Class name.
+     */
+    public static String resourceNameToClassName(String rsrcName) {
```

Comment:
MInor: please consider using newly introduced method whenever possible.
org.apache.ignite.internal.managers.deployment.GridDeploymentCommunication#processResourceRequest
org.gridgain.internal.h2.test.unit.TestClearReferences#findClasses
org.apache.ignite.testframework.GridTestExternalClassLoader#resNameToClassName (not the same though)
org.apache.ignite.testframework.GridJarClassLoader#loadClassBytes
etc

Reply:
I see this never used.
I will simply remove it from patch.